docs/product/dev-toolbar/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Dev Toolbar
+sidebar_order: 120
+description: "Bring critical Sentry insights and tools directly into your web app for easier troubleshooting with the Dev Toolbar."
+---
+
+<Alert level="info">
+  The Dev Toolbar is currently in **beta**. Beta features are still in progress and may have bugs. Please reach out on
+  [GitHub](https://github.com/getsentry/sentry-toolbar/issues) if you have any feedback or concerns.
+</Alert>
+
+Developers spend a lot of time troubleshooting their web apps on local, staging, and production environments. The Sentry Dev Toolbar pulls data from [sentry.io](http://sentry.io) and surfaces relevant actionable issues to you when you have the most context for understanding it: as you browse your own site. The Dev Toolbar is a floating widget in your web app, offering meaningful Sentry insights for the specific page being viewed through three different **page-aware** panels: Issues, Feedback, and Feature Flags.
+
+![Collapsed Dev Toolbar in context of a website](./img/toolbar-collapsed.png)
```

Comment:
I would say if possible this could even be a gif instead, so that the idea is to emphasize that it shouldn't get in the way of you browsing your site, but when you hover over it opens up (or is it when you click?)

Reply:
It opens on hover! unfortunately I don't know how to make a gif